packages/chaire-lib-backend/src/utils/processManagers/OSRMServicePath.ts

```diff
@@ -11,7 +11,8 @@ import { RoutingMode } from 'chaire-lib-common/lib/config/routingModes';
 
 const defaultDirectoryPrefix = process.env.OSRM_DIRECTORY_PREFIX
     ? process.env.OSRM_DIRECTORY_PREFIX
-    : Preferences.get('osrmRouting.directoryPrefix', '');
+    : // TODO: Migrate this value from preferences to config like the osrm modes
+    Preferences.get('osrmRouting.directoryPrefix', '');
```

Comment:
Just a thought for the future, I think that this can go into the new Engine OsrmConfig that you just added. 
Currently we have one global path for all, but I don't see why we could not have a configuration for each. (but maybe we can still keep a default global one. But that might be superfluous